microk8s-resources/actions/disable.kubeflow.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -eu
+
+source $SNAP/actions/common/utils.sh
+
+
+CONTROLLER="uk8s"
+MODEL="kubeflow"
+
+echo "Removing Kubeflow"
+"$SNAP/microk8s-juju.wrapper" destroy-controller $CONTROLLER --destroy-all-models --destroy-storage || true
```

Comment:
This could be "Destroy the kubeflow model. If there are no more models on this controller, remove the controller as well."

Also not sure about the `|| true` why would we want this?

Reply:
The `|| true` was handy for local development for me, where I might be wanting to tear everything down and not care if one destruction step failed, but I don't think we need it here (same for the `|| true` below).

If we start getting errors from these lines due to say partial initialization that makes things weird, it may be worth adding a `--force` flag that ignores any errors.